test/jdk/javax/crypto/KEM/RSA_KEM.java

```diff
@@ -122,6 +123,14 @@ public static void main(String[] args) throws Exception {
         }
     }
 
+    // To bypass the JCE security provider signature check
+    private static KEM getKemImpl(Provider p) throws Exception {
+        var ctor = KEM.class.getDeclaredConstructor(
```

Comment:
How about creating it this way only if `java.runtime.name` system property does not contain "OpenJDK"?

Reply:
True, although in that case the worse that would happen is that they use reflection instead of calling `KEM.getInstance`. Actually I am ok with this code now - as long as we have other tests that test `KEM.getInstance`.